src/layouts/shared/page-header.tsx

```diff
@@ -10,6 +10,7 @@ export default function PageHeader() {
   const { t } = useTranslation();
   const [isDarkMode] = useDarkMode();
   const { fylkeskommune, urls } = getOrgData();
+  const displayContactPage = process.env.NEXT_PUBLIC_DISPLAY_CONTACT_PAGE;
```

Comment:
This makes me think we need a second env variable, i.e. the pureservice-connect API URL. I don't think that's necessary, and we could rather check "if the contact form API URL is defined, show a link to the contact page". So I suggest renaming this env variable. 

Also, is it necessary for it to be public (`NEXT_PUBLIC_`)?

Reply:
Updated the variable name to be less confusing✅

I also agree that the .env variable can be set on the server, and that it is therefore not necessary to make the variable available from the client side (NEXT_PUBLIC_). But this can be resolved later. At the moment, I prioritize getting the whole flow in place from being able to register feedback through the front-end and sending it in to pureService.